napari/_tests/test_viewer.py

```diff
@@ -164,18 +164,23 @@ def test_screenshot(make_napari_viewer):
     assert screenshot.ndim == 3
 
 
-def test_changing_theme(make_napari_viewer):
+def test_changing_theme(make_napari_viewer, qtbot):
     """Test changing the theme updates the full window."""
-    viewer = make_napari_viewer()
+    viewer = make_napari_viewer(show=False, welcome=False)
     viewer.add_points(data=None)
-    assert viewer.theme == 'dark'
+    qtbot.wait(2000)
```

Comment:
these are some hefty wait times... did you try `processEvents()`?

Reply:
Oops those were for testing.